File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/StartProcessingJobRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class StartProcessingJobRequest {
    @JsonProperty("CountUnitOfMeasure")
    public String countUnitOfMeasure;
    @JsonProperty("JobName")
    public String jobName;
    @JsonProperty("JobType")
    public String jobType;
    @JsonProperty("Packages")
    public List<Object> packages;
    @JsonProperty("StartDate")
    public String startDate;
    @JsonProperty("VolumeUnitOfMeasure")
    public String volumeUnitOfMeasure;
    @JsonProperty("WeightUnitOfMeasure")
    public String weightUnitOfMeasure;
    public static class StartProcessingJobRequestPackagesItem {
    @JsonProperty("Label")
    public String label;
    @JsonProperty("Quantity")
    public Integer quantity;
    @JsonProperty("UnitOfMeasure")
    public String unitOfMeasure;
}
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/StartProcessingJobRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class StartProcessingJobRequest(
    @JsonProperty("CountUnitOfMeasure")
    val countUnitOfMeasure: String? = null,
    @JsonProperty("JobName")
    val jobName: String? = null,
    @JsonProperty("JobType")
    val jobType: String? = null,
    @JsonProperty("Packages")
    val packages: List<Any>? = null,
    @JsonProperty("StartDate")
    val startDate: String? = null,
    @JsonProperty("VolumeUnitOfMeasure")
    val volumeUnitOfMeasure: String? = null,
    @JsonProperty("WeightUnitOfMeasure")
    val weightUnitOfMeasure: String? = null
)
